homeassistant/components/sensor/bom_weather_current.py

```diff
@@ -0,0 +1,188 @@
+"""
+Support for bom.gov.au current condition weather service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.bom_weather_current
+"""
+
+import logging
+import requests
+import voluptuous as vol
+import datetime
```

Comment:
Can you please update the ordering of the imports according [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)?